io-ballerina/build.gradle

```diff
@@ -109,111 +112,112 @@ task initializeVariables {
 
     gradle.taskGraph.whenReady { graph ->
         if (graph.hasTask(":${packageName}-ballerina:build") || graph.hasTask(":${packageName}-ballerina:publish") ||
-                graph.hasTask(":${packageName}-ballerina:publishToMavenLocal"))  {
-            ballerinaTest.enabled = false
+                graph.hasTask(":${packageName}-ballerina:publishToMavenLocal")) {
+            needSeparateTest = false
+
+            if (graph.hasTask(":${packageName}-ballerina:build")) {
+                needBuildWithTest = true
+            }
+            if (graph.hasTask(":${packageName}-ballerina:publish")) {
+                needBuildWithTest = true
+                needPublishToCentral = true
+            }
+            if (graph.hasTask(":${packageName}-ballerina:publishToMavenLocal")) {
```

Comment:
Shall we publish the ballerina package to the local repository with this? Or shall we add a separate parameter for that?

Reply:
Since it is an explicit call, I think better to add a custom argument. For example, `-PpublishToLocalCentral`. WDYT?

Reply:
+1

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/133/commits/1c1383f486376e07f6af328711a9280f1e98014d